src/function.rs

```diff
@@ -667,6 +668,16 @@ fn uuid(_context: Context) -> FunctionResult {
   Ok(uuid::Uuid::new_v4().to_string())
 }
 
+fn which_exec(_context: Context, s: &str) -> FunctionResult {
+  let path = which::which(s).unwrap_or_default();
```

Comment:
This will swallow the error from `which`. But I checked out the `which` crate, and it looks like it will actually swallow I/O errors, which (haha) isn't ideal. I opened an error on the crate: https://github.com/harryfei/which-rs/issues/100

I don't think this is a show-stopper, since at the very worst, we can swap out the implementation later.